src/lightning/fabric/wrappers.py

```diff
@@ -185,6 +189,28 @@ def __getattr__(self, item: Any) -> Any:
             self._validate_method_access(item, attr)
             return attr
 
+    def __setattr__(self, name: str, value: Any) -> None:
+        if not self._fabric_module_initialized:
+            return super().__setattr__(name, value)
+
+        # Get the _original_module attribute
+        original_module = self._original_module
+        original_has_attr = hasattr(original_module, name)
+        # Can't use super().__getattrr__ because nn.Module only checks _parameters, _buffers, and _modules
+        fabric_has_attr = name in self.__dict__
```

Comment:
Can't this be?

```suggestion
        fabric_has_attr = hasattr(self, name)
```

Reply:
because we have overridden getattr ourselves, and this line here was to avoid calling getattr.